bin/deployment/deploy-project.php

```diff
@@ -30,26 +31,24 @@
     die;
 }
 
-
 if (!empty($_POST['install'])) {
     $projectDeployer
         ->setDatabaseCredentials($_POST['database_host'], $_POST['database_user'], $_POST['database_password'], $_POST['database_name'], $_POST['database_test_name'])
         ->setEnvironment($_POST['environment'])
         ->setSecureRoutes(isset($_POST['secure']))
         ->deploy();
 
-    header('content-type:text/plain');
     $numErrors = count($projectDeployer->getErrors());
     if ($numErrors > 0) {
+        $errorMessages = [];
         foreach ($projectDeployer->getErrors() as $error) {
-            echo $error . "\n";
+            $errorMessages[] = $error . "\n";
         }
-        echo 'There were ' . ($numErrors > 1 ? $numErrors . ' errors' : $numErrors . ' error');
+        latte('deploy', ['errorMessages' => $errorMessages]);
```

Comment:
Please just initialize the errorMessages variable there, and call a single `latte('deploy', $args)` at the bottom of this file